.circleci/config.yml

```diff
@@ -11,7 +11,9 @@ jobs:
     steps:
       - checkout
       - python/load-cache
-      - python/install-deps
+      - run:
```

Comment:
We could use https://circleci.com/developer/orbs/orb/circleci/python?version=0.3.2#commands-install-deps with specified file name